### DISCUSSION TITLE
Is it possible to start the app minimized?

### DISCUSSION THREAD

**Discussion by WhatsAllThisThen** [localsend/localsend#594] [Discussion (English, International)] [answered]
I just started using Localsend, and it's working really well. I have one question though. I'm using Linux, and I've included Localsend in the Startup Applications list. It does start up, but then I have to close it manually (minimize to tray is enabled.) Is there a way to have it start up minimized in the first place?

**Comment by Tienisto** [comment, marked as answer]
You need to add the `autostart` parameter when running the executable: https://github.com/localsend/localsend/issues/529

You also need to find the config file and set `"flutter.ls_auto_start_launch_minimized": true`. The config file should be located in `XDG_DATA_HOME` according to the Flutter documentation.

Another solution is to compile LocalSend from source because this should be supported in the `main` branch (but unreleased yet)

**Reply by WhatsAllThisThen** [reply]
Thank you!

Adding autostart was all I needed to do as it turned out. No need to mess with the config file, which is a good thing because it looks scary! 

![image](https://github.com/localsend/localsend/assets/140795122/78d8bd8c-05f7-446c-8e3a-6616e792fdf7)

**Comment by chenshaoju** [comment]
On the Windows system, you need to add the ```--hidden``` parameter, for example (I am using the portable version):

``` "D:\Software\LocalSend\localsend_app.exe" --hidden ```

**Comment by nullman** [comment]
The --hidden flag also works in Linux (Arch):

`localsend --hidden`

**Reply by kINo204** [reply]
Really helps!
